instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/sessionreplay/BaseSessionReplayActivity.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sdk.integration.sessionreplay
+
+import android.app.Activity
+import android.graphics.Point
+import android.os.Build
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.datadog.android.Datadog
+import com.datadog.android.rum.Rum
+import com.datadog.android.rum.tracking.ActivityViewTrackingStrategy
+import com.datadog.android.sdk.integration.R
+import com.datadog.android.sdk.integration.RuntimeConfig
+import com.datadog.android.sdk.utils.getForgeSeed
+import com.datadog.android.sdk.utils.getSessionReplayPrivacy
+import com.datadog.android.sdk.utils.getSrSampleRate
+import com.datadog.android.sdk.utils.getTrackingConsent
+import com.datadog.android.sessionreplay.SessionReplay
+import com.datadog.android.sessionreplay.SessionReplayConfiguration
+import com.datadog.android.sessionreplay.SessionReplayPrivacy
+import java.util.Random
+
+internal abstract class BaseSessionReplayActivity : AppCompatActivity() {
+    @Suppress("CheckInternal")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val config = RuntimeConfig.configBuilder().build()
+        val trackingConsent = intent.getTrackingConsent()
+        val sessionReplayPrivacy = intent.getSessionReplayPrivacy()
+        val sessionReplaySampleRate = intent.getSrSampleRate()
+        Datadog.setVerbosity(Log.VERBOSE)
+        val sdkCore = Datadog.initialize(this, config, trackingConsent)
+        checkNotNull(sdkCore)
+        val featureActivations = mutableListOf(
+            // we will use a large long task threshold to make sure we will not have LongTask events
+            // noise in our integration tests.
```

Comment:
since we don't check RUM data, it is probably not needed.

Reply:
yup was a leftover